### ISSUE TITLE
Meeting Media Manager BUG

### ISSUE THREAD

**Issue by HananiasSQ** [sircharlo/meeting-media-manager#2354]
### What happened?

On macOS Sonoma in my MacBook Pro 14, When I start sharing the MMM secondary screen in Zoom, verification to allow use of microphone alert keeps popping up countless times on Meeting Media Manager. That stops the media from playing as well. So it delays streaming the songs and videos on Zoom. Is there something wrong?
System Preferences already has MMM allow to use Microphone. 

### To Reproduce

3. Do '....'

### What did you expect to happen?

I have no idea why

### Version

latest version as of 2024.2.4

### What platform are you seeing the problem on?

MacOS

### Relevant log output

```shell
No log
```

### Screenshots

![Screen Shot 2024-02-04 at 7 14 25 PM](https://github.com/sircharlo/meeting-media-manager/assets/158812402/ab15134d-3fd0-4e0f-8116-38e78ff16b6b)

**Comment by mtdvlpr** [comment]
Hi, please see #2178 for possible solutions

**Comment by HananiasSQ** [comment]
Thank you brother! That did it! It only asks one time and that is it. This is such a huge help. 
Because I was in need of an upgrade, and thought the MacBook Pro was a safe and reliable option. 
You are a life saver! Specially with the Memorial coming up soon I was rehearsing how to project the songs on to a screen with the MacBook and that's when I discovered that JW library doesn't support external monitor on the MacBook.
Found M3 and loved the idea, but that bug was had me concerned for a moment. 
The steps I took to fix it was to:
- uncheck the allow use of the microphone. 
- Change the name "Meeting Media Manager" to "M3" (I went with the common nickname)
- Put the terminal command found in post #2178 
- Restart M3 and Zoom
- while starting the share screen option, allow use of microphone... and that is it! 
No more alert pop ups!
Thank you guys so much!
